modules/op-release-notes-1-8.adoc

```diff
@@ -50,8 +238,141 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
+// https://github.com/tektoncd/cli/pull/1542
+// Piyush Garg @piyush-garg
+
+* Before this update, if you ran a `tkn pr delete --ignore-running` command and a pipeline run did not have a `status.condition` value, the `tkn` CLI tool produced a null-pointer error (NPE). This update fixes the issue so that the CLI tool does generate an error and correctly ignores pipeline runs that are still running.
+// https://github.com/tektoncd/cli/pull/1532
+// PRADEEP KUMAR @pradeepitm12
+
+* Before this update, if you used the `tkn pr delete --keep <value>` or `tkn tr delete  --keep <value>` commands, and the number of pipeline runs or task runs was less than the value, the command did not return an error as it should. This update fixes the issue so that the command correctly returns an error under those conditions.
+// https://github.com/tektoncd/cli/pull/1541
+// Shiv Pratap Verma @pratap0007
+
+* Before this update, if you used the `tkn pr delete` or `tkn tr delete` commands with the `-p` or `-t` flags together with the `--ignore-running` flag, the commands incorrectly deleted running or pending resources. This update fixes the issue so that these commands correctly ignore running or pending resources.
+// https://github.com/tektoncd/cli/pull/1578
+// Piyush Garg @piyush-garg
+
+// .Operator
+
+* With this update, you can configure {tekton-chains} by using the `TektonChain` custom resource. This feature enables your configuration to persist across upgrades, unlike the `chains-config` config map, which gets overwritten during upgrades.
+// no link provided
+
+* With this update, `ClusterTask` resources no longer run as root by default, except for the `buildah` and `s2i` cluster tasks.
+// https://issues.redhat.com/browse/SRVKP-2385
+// Shubham Minglani
+
+* Before this update, tasks on {pipelines-title} 1.7.1 failed when using `init` as a first argument followed by two or more arguments. With this update, the flags are parsed correctly, and the task runs are successful.
+// https://issues.redhat.com/browse/SRVKP-2340 (borrowed from 1.7.2 release notes)
+// Vincent Demeester @vdemeester
+
+* Before this update, installation of the {pipelines-title} Operator on {product-title} 4.9 and 4.10 failed due to an invalid role binding, with the following error message:
++
+[source,terminal]
+----
+error updating rolebinding openshift-operators-prometheus-k8s-read-binding: RoleBinding.rbac.authorization.k8s.io "openshift-operators-prometheus-k8s-read-binding" is invalid: roleRef: Invalid value: rbac.RoleRef{APIGroup:"rbac.authorization.k8s.io", Kind:"Role", Name:"openshift-operator-read"}: cannot change roleRef
+----
++
+This update fixes the issue so that the failure no longer occurs.
+// (borrowed from 1.7.2 release notes) https://issues.redhat.com/browse/SRVKP-2327
+// Khurram Baig
```

Comment:
@khrm  Please review and approve. Thanks.

Reply:
looks good